[ 2025-08-19 23:31:25,014 ] 1025 httpx - INFO - HTTP Request: GET https://generativelanguage.googleapis.com/v1beta/models/gemini-2.5-pro "HTTP/1.1 200 OK"
[ 2025-08-19 23:31:31,742 ] 1025 httpx - INFO - HTTP Request: POST https://generativelanguage.googleapis.com/v1beta/models/text-embedding-004:batchEmbedContents "HTTP/1.1 200 OK"
[ 2025-08-19 23:32:32,569 ] 1025 httpx - INFO - HTTP Request: GET https://generativelanguage.googleapis.com/v1beta/models/gemini-2.5-pro "HTTP/1.1 200 OK"
[ 2025-08-19 23:32:39,809 ] 1025 httpx - INFO - HTTP Request: POST https://generativelanguage.googleapis.com/v1beta/models/text-embedding-004:batchEmbedContents "HTTP/1.1 200 OK"
[ 2025-08-19 23:32:43,058 ] 1025 httpx - INFO - HTTP Request: POST https://generativelanguage.googleapis.com/v1beta/models/text-embedding-004:batchEmbedContents "HTTP/1.1 200 OK"
[ 2025-08-19 23:32:44,000 ] 6418 google_genai.models - INFO - AFC is enabled with max remote calls: 10.
[ 2025-08-19 23:32:47,449 ] 1025 httpx - INFO - HTTP Request: POST https://generativelanguage.googleapis.com/v1beta/models/gemini-2.5-pro:generateContent "HTTP/1.1 500 Internal Server Error"
[ 2025-08-19 23:32:47,452 ] 65 llama_index.llms.google_genai.utils - WARNING - Retrying llama_index.llms.google_genai.base.GoogleGenAI._chat in 1.0 seconds as it raised ServerError: 500 INTERNAL. {'error': {'code': 500, 'message': 'An internal error has occurred. Please retry or report in https://developers.generativeai.google/guide/troubleshooting', 'status': 'INTERNAL'}}.
[ 2025-08-19 23:32:48,453 ] 6418 google_genai.models - INFO - AFC is enabled with max remote calls: 10.
[ 2025-08-19 23:32:49,186 ] 1025 httpx - INFO - HTTP Request: POST https://generativelanguage.googleapis.com/v1beta/models/gemini-2.5-pro:generateContent "HTTP/1.1 500 Internal Server Error"
[ 2025-08-19 23:32:49,188 ] 65 llama_index.llms.google_genai.utils - WARNING - Retrying llama_index.llms.google_genai.base.GoogleGenAI._chat in 1.051576076526608 seconds as it raised ServerError: 500 INTERNAL. {'error': {'code': 500, 'message': 'An internal error has occurred. Please retry or report in https://developers.generativeai.google/guide/troubleshooting', 'status': 'INTERNAL'}}.
[ 2025-08-19 23:32:50,239 ] 6418 google_genai.models - INFO - AFC is enabled with max remote calls: 10.
[ 2025-08-19 23:32:56,778 ] 1025 httpx - INFO - HTTP Request: POST https://generativelanguage.googleapis.com/v1beta/models/gemini-2.5-pro:generateContent "HTTP/1.1 200 OK"
[ 2025-08-19 23:32:56,779 ] 6429 google_genai.models - INFO - AFC remote call 1 is done.
